ccd-definition/ComplexTypes/HearingBooking.json

```diff
@@ -21,7 +21,8 @@
     "LiveFrom": "01/01/2017",
     "ID": "HearingBooking",
     "ListElementCode": "venue",
-    "FieldType": "Text",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "HearingVenue",
```

Comment:
when we go from FixedList -> DynamicList, is this data an easy swap or will it require migration to work. Maybe "if we swap..." is a better q 😂